packages/flutter_test/lib/src/controller.dart

```diff
@@ -24,6 +24,17 @@ const double kDragSlopDefault = 20.0;
 
 const String _defaultPlatform = kIsWeb ? 'web' : 'android';
 
+// This is to prevent `SemanticsController.find` from shadowing `finders.dart#find`.
+const CommonFinders _find = find;
+
+/// The action done on every visited node when using the
+/// [SemanticsController._traverse] method.
+///
+/// The return value determines whether the traversal is finished or not, so
+/// returning `true` will short circuit the traversal at the current `node`.
+typedef _VisitSemanticsNodeCallback = bool Function(SemanticsNode node);
```

Comment:
This already exists as `SemanticsNodeVisitor` in the framework. Can we just reuse that?

Reply:
(Although, it uses the opposite meaning for the return value, might be worthwhile to standardize.)

Reply:
This was removed per the "`_traversal` didn't need modification" discussion below.